pinot-core/src/test/java/org/apache/pinot/core/query/aggregation/groupby/GroupByTrimTest.java

```diff
@@ -130,7 +129,7 @@ void testGroupByTrim(QueryContext queryContext, int minSegmentGroupTrimSize, int
     // Extract the execution result
     List<Pair<Double, Double>> extractedResult = extractTestResult(resultsBlock.getTable());
 
-    assertEquals(extractedResult, expectedResult);
+    Assert.assertEquals(extractedResult, expectedResult);
```

Comment:
(nit) We usually use static import for `Assert` in test

Reply:
I had changed this due to checkstyle violations:

```[WARNING] (imports) AvoidStaticImport: Using a static member import should be avoided```

I can revert back so long as checkstyle still passes